site/docs/operations/kubernetes.md

```diff
@@ -43,27 +43,36 @@ enterprise versions also for Hazelcast Jet Management Center.
 
 ### Installing the Chart
 
+Hazelcast Official Chart Repository is hosted at https://github.com/hazelcast/charts
```

Comment:
This confused me that the charts repo moved away from AWS, but it is actually where the **sources** are hosted. So maybe word this as 

>Hazelcast Official Chart Repository has sources hosted at https://github.com/hazelcast/charts

or similar.